lib/portlayer/network/context.go

```diff
@@ -912,6 +953,24 @@ func (c *Context) DeleteScope(name string) error {
 		return fmt.Errorf("scope has bound endpoints")
 	}
 
+	// remove gateway ip from bridge interface
+	link, err := getBridgeLink()
```

Comment:
Should we stash this somewhere on initialization? It seems like something that should be done during portlayer init and likely a fatal failure (without it we cannot do name resolution for containers or port forwarding).